.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
-## This PR is a:
+## Scope of this PR
 
-- [ ] New topic
-- [ ] Content update
-- [ ] Content fix or rewrite
-- [ ] Bug fix or improvement
+We set labels (and therefore points) based on the scope of changes with a PR.
```

Comment:
Could cut down further a bit if you wanted, linking to the contributor guide of points. 

```suggestion
We set [labels](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs) based on the scope of changes with a PR.
```